molecule/additional_flags/molecule-no-systemd.yml

```diff
@@ -0,0 +1,74 @@
+---
```

Comment:
If `molecule.yml` is changed this file would no longer be needed.

Reply:
Please see commit 2e1b063.